src/main/java/seedu/foodrem/model/item/Item.java

```diff
@@ -142,14 +142,14 @@ public boolean equals(Object other) {
         }
 
         Item otherItem = (Item) other;
-        return otherItem.getName().equals(name)
```

Comment:
Not a big problem, just wanted to find out why this was changed?

Reply:
Since this is internally called from the class itself, just get the private field value directly, no need to use an accessor method. Cleaner code, better practice, and also more efficient (no need to create another stack frame for the method call).